configs/relengworker-prod/config.yml

```diff
@@ -210,3 +210,59 @@ worker_types:
         sla_seconds: 120
         capacity_ratio: 1.0
         min_replicas: 1
+
+  - worker_type: xpi-t-signing
+    provisioner: scriptworker-k8s
+    root_url: "https://firefox-ci-tc.services.mozilla.com"
+    deployment_namespace: prod-signing
+    deployment_name: signing-prod-relengworker-firefoxci-signing-t-1
+    autoscale:
+      algorithm: sla
+      args:
+        max_replicas: 10
+        avg_task_duration: 60
+        sla_seconds: 120
+        capacity_ratio: 1.0
+        min_replicas: 1
```

Comment:
Can you set `min_replicas` to 0 everywhere please. Just discussed this on slack, and I'm going to adjust other workers too.

Reply:
Done. Also added the dev instances.